Platform/ApollolakeBoardPkg/Library/Stage2BoardInitLib/Stage2BoardInitLib.c

```diff
@@ -670,27 +689,14 @@ BoardInit (
     RegisterHeciService();
     InitPlatformService ();
 
-    // Lock down SPI for all other payload entry except FWUpdate and OSloader
-    // as this phase is too early for them to lock it here
-    SpiBaseAddress = GetDeviceAddr (OsBootDeviceSpi, 0);
-    SpiBaseAddress = TO_MM_PCI_ADDRESS (SpiBaseAddress);
-
-    if ((GetBootMode() != BOOT_ON_FLASH_UPDATE) && (GetPayloadId() != 0) && (GetPayloadId() != UEFI_PAYLOAD_ID_SIGNATURE)) {
-      // Set the BIOS Lock Enable and EISS bits
-      MmioOr8 (SpiBaseAddress + R_SPI_BCR, (UINT8) (B_SPI_BCR_BLE | B_SPI_BCR_EISS));
-
-      ClearFspHob ();
+    if (GetPayloadId() != 0) {
```

Comment:
Don't you have to keep it open for FLASH_UPDATE boot mode?

Reply:
Both Osloader and FirmwareUpdate PayloadId is 0. So we don't need check firmware update separately.